actions/terra-k8s-linter/policy/deployments/deny.rego

```diff
@@ -0,0 +1,32 @@
+package main
+
+import data.kubernetes
+
+
+name = input.metadata.name
```

Comment:
`name` feel ambiguous here, what object is this name for? The app name (agora?) some other name? I'd either use a more descriptive name e.g. terra_app_name and/or add a short  comment to explain.

Reply:
metadata.name is the name of the manifest. I'll put a comment. However, this is something I would expect a developer to know even with little experience with Kubernetes/GO

Reply:
aha, cool, thanks!

Reply:
33cc7c9